docs/source/hackrf_tools.rst

```diff
@@ -0,0 +1,140 @@
+============
+HackRF Tools
+============
+
+Great Scott Gadgets provides some commandline tools for interacting with HackRF. For information on how to use each tool look at the help information provided via commandline (e.g. ``hackrf_transfer -h``) or the `manual pages <http://manpages.ubuntu.com/manpages/utopic/man1/hackrf_info.1.html>`__.
```

Comment:
The manual pages referred to are extremely out of date - I don't think it's helpful to link to them here.